Add codebooks for sdg 2, 3 and 4 (#19)
nestauk/sdg-mapping: codebooks/codebook_sdg_3.md

# Goal 3. Good Health and Well-Being
**Ensure healthy lives and promote well-being for all at all ages**

## Labelling Instructions

### Key Concepts of Goal 3

To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:

> - Improving the health of millions

Reviewer: ```suggestion
- Improving human health

```

> - Increase life expectancy and reduce deaths caused by external factors such as road traffic accidents

Reviewer: ```suggestion
- Increase life expectancy 
- Infant mortality
- Reduce deaths caused by external factors e.g.. road traffic accidents

```

> - Reduce the global maternal mortality ratio

Reviewer: ```suggestion
- Reduce the global maternal mortality rate

```

Reviewer: I know this goal says 'ratio' but I think out of context 'rate' makes more sense

- Reduce the number deaths and illnesses from hazardous chemicals and air, and water and soil pollution*

> - End the epidemics of diseases and premature deaths caused by such diseases

Reviewer: ```suggestion
- Epidemics and related premature deaths
- Communicable and non-communicable diseases
- Mental health, well-being and suicide
```

> - Prevention and treatment of substance abuse

Reviewer: ```suggestion
- Prevention and treatment of substance abuse (incl. tobacco)

```

- Support of the research and development of vaccines and medicines
- Universal healthcare and universal access to sexual and reproductive health-care services*

> - Access to health-care services*

Reviewer: ```suggestion
- Access to health-care services*
- Universal health coverage, health insurance and access to medicines
- Development, training and retention of the health workforce
- Early warning, risk reduction and management of national/international health risks
```

> To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of these. More detail on the specific targets is provided below.

Reviewer: ```suggestion
```

> * There are specific goals for consequences of consumption and production, gender and reducing inequality. Please only label projects `Yes` if they mainly relate to these areas in the context of state spending.

Reviewer: ```suggestion
* There are specific goals for broader gender inequality and other inequalities. Please only label projects `Yes` if they concern equality in a health-specific context.

```

### Key terms (non-exhaustive)

health, wellbeing, life quality, health risks, prevention, disease, illness, health-related, screening, therapy, disorders, healthcare services, drug discovery, pharmaceutical, cancer, reproductive health, morality rate, welfare

### Not to be confused with

- 10. Reduced Inequalities
  - Access to quality services for all not specifically related to health

> - 12. Responsible Consumption and Production
>   - Concerns the impact of chemicals and waste on not only human health but also environmental health

Reviewer: ```suggestion
```

Remember, many projects will cover more than one Goal, but to be tagged as Goal 3 they must include at least one of the Key Concepts above. Further information on other goals can be found [here](https://unstats.un.org/sdgs/metadata/).

For example:

| Project description                                                                        | Is Goal 3? |
|--------------------------------------------------------------------------------------------|------------|
| Project discussing new techniques for creating prostheses for deafness                                     | Yes        |

> | Project discussing imagining strategies on patients with intermediate risk of coronary artery disease                                    | Yes        |

Reviewer: imaging? (instead of imagining)

Author: ```suggestion
| Project discussing imaging strategies on patients with intermediate risk of coronary artery disease                                    | Yes        |

```

| Project discussing clot formation as a potential diagnostic tool for Alzheimer's disease                                     | Yes        |
| Project discussing a early-warning system for agriculture-related diseases such as wheat rust diseases                                    | No         |

> | Project discussing metabolic interactions in oceanic photosymbioses                                    | No         |

Reviewer: ```suggestion
| Project discussing metabolic interactions in oceanic ecosystems                                    | No         |

```
